charts/pulsar/values.yaml

```diff
@@ -963,6 +963,8 @@ pulsar_manager:
     hostname: ""
     path: "/"
 
+  ## If set use existing secret with specified name to set pulsar admin credentials.
+  existingSecret:
```

Comment:
Can you name it as `existingSecretName`?

Reply:
Updated. It was my opinio the original name is the most used for this purpose and thus best fit into the helm charts ecosystem, but this of course might be only mi bias, sinc I does not have any strong support for this..